client/src/views/nodes.js

```diff
@@ -6,15 +6,15 @@ import {defaultSortInfo} from '../components/sorter';
 import NodeStatusChart from '../components/nodeStatusChart';
 import api from '../services/api';
 import test from '../utils/filterHelper';
-import NodesPanel from '../components/nodesPanel';
+import NodesPanel, {getReadyStatus} from '../components/nodesPanel';
```

Comment:
This PR looks awesome! Thanks once again for your contribution :)

I would like to request one small change. I don't love the idea of exporting utility functions like this from other components. Instead, let's create a new `utils/nodeHelpers.js` file, move `getReadyStatus` inside there, and then import that in both `components/nodesPanel.js` and `views/nodes.js`.

Thoughts?

Reply:
Yes, sure ;)